opm/simulators/linalg/GraphColoring.hpp

```diff
@@ -19,91 +19,82 @@
 #ifndef OPM_GRAPHCOLORING_HEADER_INCLUDED
 #define OPM_GRAPHCOLORING_HEADER_INCLUDED
 
```

Comment:
All changes in this file before ling 214 are caused by clang-format, and are not actually edited for this PR